mqtt-bridge/.gitignore

```diff
@@ -0,0 +1,17 @@
+/target
+
+# Eclipse specific
+.project
+.settings/
+.prefs
+.classpath
+.apt_generated/
+.apt_generated_tests/
+
+# IntelliJ IDEA specific
+.idea/
+*.iml
+
+# VS Code
+.factorypath
+.vscode
```

Comment:
I would copy the `.gitignore` file of the strimzi-kafka-bridge (HTTP bridge) [1]

And place this file in the parent directory


[1] https://github.com/strimzi/strimzi-kafka-bridge/blob/main/.gitignore

Reply:
done as you requested.